README.md

```diff
@@ -79,9 +106,9 @@ Creating a task manifest file.
                     "plugin_name": "statistics",
 		    "config":
     			{
-	    			"SlidingWindowLength": 15
+	    			"slidingWindowLength": 5,
+                    "slidingFactor": 2,
```

Comment:
can you elaborate what's recommended "slidingwindorlength" and "slidingfactor" in terms of the data set size and the reasonings behind your recommendation.